src/java/org/apache/cassandra/utils/FBUtilities.java

```diff
@@ -865,6 +865,49 @@ public static String prettyPrintMemory(long size)
         return prettyPrintMemory(size, "");
     }
 
+    /**
+     * Formats a latency value in milliseconds for display, appending an "ms" suffix.
+     * The formatted output is rounded to three decimal places.
+     * For example, "5000.000 ms", "100.000 ms", "0.050 ms", "0.000 ms", "NaN ms".
+     * @param latency   Latency in milliseconds to print.
+     */
+    public static String prettyPrintLatency(double latency)
+    {
+        return String.format("%.3f ms", latency);
+    }
+
+    /**
+     * Formats a ratio value for display, rounds it to three decimal places.
+     * For example, "10.000", "1.000", "0.050", "0.001", "0.000", "NaN".
+     * @param ratioObj   Ratio to print.
+     */
+    public static String prettyPrintRatio(Object ratioObj)
```

Comment:
prettyPrintRatio(Object ratioObj) signature could be changed to prettyPrintRatio(Number ratio), but I can't completely eliminate the ifs. And actually, I'd rather stick with prettyPrintRatio(Object ratioObj).
Here's why: String.format raises an exception when format specifiers don't match the format arguments like in:
```
jshell> String.format("%.3f ms", (Number)123)
|  Exception java.util.IllegalFormatConversionException: f != java.lang.Integer 
```
And then there is also the "null" case.

So because sstableCompressionRatio and bloomFilterFalseRatio are untyped, we have two options:
1. Branch type casting
2. Handle IllegalFormatConversionException
And the third option is to refactor StatsTable.sstableCompressionRatio and StatsTable.bloomFilterFalseRatio to be typed as "double". I assume that the third option is off-the-table since it might result in too many changes.

Reply:
@zaaath could you please try the third option?

Reply:
@smiklosovic sure, working on that.

Reply:
@smiklosovic completed, refactored StatsTable.sstableCompressionRatio and StatsTable.bloomFilterFalseRatio to be of the "double" type instead of "Object"

Is there anything else that needs to be handled?